MiniApp/Classes/core/Storage/MiniAppCacheVerifier.swift

```diff
@@ -1,27 +1,26 @@
 import CommonCrypto
-import RSDKUtils
 
 internal class MiniAppCacheVerifier {
-    let keystore = KeyStore()
+    let miniAppVerificationStore = MiniAppVerificationStorage()
 
     private func generateKeyId(for appId: String, version: String) -> String {
         "\(appId) | \(version)"
     }
 
     func verify(appId: String, version: String) -> Bool {
-        if keystore.key(for: appId) != nil {
+        if miniAppVerificationStore.getCacheVerificationInfo(for: appId) != nil {
             return false // v2.5 legacy cleaning
         }
         let cachedFilesHash = calculateHash(appId: appId, version: version)
-        let storedHash = keystore.key(for: generateKeyId(for: appId, version: version)) ?? ""
+        let storedHash = miniAppVerificationStore.getCacheVerificationInfo(for: generateKeyId(for: appId, version: version)) ?? ""
         return cachedFilesHash == storedHash
     }
 
     func storeHash(for appId: String, version: String) {
-        keystore.removeKey(for: appId) // v2.5 legacy cleaning
+        miniAppVerificationStore.removeCacheInfo(for: appId) // v2.5 legacy cleaning
```

Comment:
I think that is something we can get rid of with next major release, as our only known customer so far is Link and they already use v2.7.

cc @corycaywood